Prace_domowe/Praca_domowa2/Wolny_Krzysztof/PD2.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>To nie jest oczywiste, że imputacja po kolei taki sam wynik jak jednoczesna imputacja dwóch cech. Jeśli potrafisz to uzasadnić, to super, a jeśli nie to jest to temat do zbadania</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>